Source/Lua/LuaBindingsEntities.cpp

```diff
@@ -763,7 +763,8 @@ LuaBindingRegisterFunctionDefinitionForType(EntityLuaBindings, MOPixel) {
 	return ConcreteTypeLuaClassDefinition(MOPixel, MovableObject)
 
 	    .property("TrailLength", &MOPixel::GetTrailLength, &MOPixel::SetTrailLength)
-	    .property("Staininess", &MOPixel::GetStaininess, &MOPixel::SetStaininess);
+	    .property("Staininess", &MOPixel::GetStaininess, &MOPixel::SetStaininess)
+	    .property("Color", &MOPixel::GetColorIndex, &MOPixel::SetColorIndex);
```

Comment:
I'd prefer to either use ColorIndex as a property or even using  Get/SetColorIndex functions instead. We need to consider how to use this when palette is gone, and so being able to add a new unpaletted Color property would be better than having to break compatibility by changing the meaning of Color from a palette index to a "real" Color

Reply:
Good point, should be an easy fix; i'll just split it up into Get/SetColorIndex then